src/liballoc/range.rs

```diff
@@ -11,142 +11,7 @@
 #![unstable(feature = "collections_range",
             reason = "waiting for dust to settle on inclusive ranges",
             issue = "30877")]
+#![rustc_deprecated(reason = "moved to core::ops", since = "1.19.0")]
```

Comment:
As this module only contained `RangeArgument` and that's been removed, this entire module can just be deleted along with its reexports in `collections` and `std::collections`.